csharp/ql/src/Security Features/CWE-016/ASPNetMaxRequestLength.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Large maxRequestLength value.
+ * @description maxRequestLength should be set as minimum as possible.
+ * @kind problem
+ * @problem.severity warning
+ * @precision very-high
+ * @id cs/web/debug-binary
+ * @tags security
+ *       maintainability
+ *       frameworks/asp.net
+ *       external/cwe/cwe-16
+ */
+
+import csharp
+from XMLAttribute a
+where
+  a.getName().toLowerCase() = "maxrequestlength" and a.getValue() > "4096"
```

Comment:
The test `a.getValue() > "4096"` actually performs a lexicographic string comparison; `a.getValue().toInt() > 4096` is better.